### DISCUSSION TITLE
Discord invite is out of date

### DISCUSSION THREAD

**Discussion by benbankes** [TanStack/query#667] [General] [answered]
If I try to join the Discord server, I am greeted by a stinky emoji.

![image](https://user-images.githubusercontent.com/2132746/86067057-1cf1e480-ba31-11ea-9c4c-3d777db38910.png)

I don't know where to put this comment, since it's not a bug, so I am documenting it here.

**Comment by MichaelDeBoey** [comment, marked as answer]
It's fixed in https://github.com/tannerlinsley/react-query/commit/2032924fbffbf937a49f7238d21b46b88c0088e1 🙂
